runtime/lua/vim/lsp/util.lua

```diff
@@ -720,18 +720,29 @@ end
 --         ignoreIfExists? bool
 function M.rename(old_fname, new_fname, opts)
   opts = opts or {}
-  local bufnr = vim.fn.bufadd(old_fname)
-  vim.fn.bufload(bufnr)
   local target_exists = vim.loop.fs_stat(new_fname) ~= nil
   if target_exists and not opts.overwrite or opts.ignoreIfExists then
     vim.notify('Rename target already exists. Skipping rename.')
     return
   end
+  local oldbuf = vim.fn.bufadd(old_fname)
```

Comment:
I'd like to standardize on prev_buf and curr_buff generally throughout the LSP client if that's ok.

Reply:
`oldUri` and `newUri` are used in the LSP structure. I don't think switching the terms midway would serve readability and would prefer to keep the names in this context

Reply:
That's fine